docs/infrastructure/microk8s-quickstart.md

```diff
@@ -80,18 +77,19 @@ Now that MicroK8s is running on the machine we can proceed to enabling the neces
     microk8s.status --wait-ready
     ```
 
-4.  Install version v2.16.3 of the Helm client:
+4.  Export your kubeconfig so that Helm knows on which cluster to install the charts:
 
     ```bash
-    sudo snap install helm --classic --channel=2.16/stable
+    microk8s.config > ~/.kube/config
     ```
 
-5.  Install the Helm server:
+5.  Install Helm version 3.2.1:
 
     ```bash
-    microk8s.kubectl create serviceaccount --namespace kube-system tiller
-    microk8s.kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
-    helm init --service-account tiller
+    curl -L "https://get.helm.sh/helm-v3.2.1-linux-amd64.tar.gz" | tar -zxf - &&
+    sudo mv linux-amd64/helm /usr/local/bin/helm &&
+    sudo chmod +x /usr/local/bin/helm &&
+    rm -r linux-amd64
```

Comment:
```suggestion
    curl -L "https://get.helm.sh/helm-v3.2.1-linux-amd64.tar.gz" | tar -zxf - linux-amd64/helm && \
    sudo chmod +x ./helm && \
    sudo mv ./helm /usr/local/bin/helm
```

Reply:
There were missing slashes in the end and we can extract just the needed files to avoid having to remove the extraction directory

Reply:
This is cleaner, yes! :+1: